lib/api/apiUtils/object/createAndStoreObject.js

```diff
@@ -19,10 +19,12 @@ function _storeInMDandDeleteData(bucketName, dataGetInfo, cipherBundle,
             if (err) {
                 return callback(err);
             }
+            let nullVersionLength;
             if (dataToDelete) {
+                nullVersionLength = dataToDelete[0].size;
```

Comment:
I didn't think of this. The main caveat is that mpus are stored as an array of multiple data locations, so you may not be able to assume the entire object's content is in `dataToDelete[0].size`. If the null version was stored as a mpu, you will need to iterate through the array and add up the size of each part (the same way a mpu's content-length was calculated). Take this with a grain of salt, I don't know this first-hand, but it's what I interpreted from referencing the code.